frontend/src/views/LabelView.vue

```diff
@@ -79,4 +84,85 @@ watchEffect(() => {
       />
     </div>
   </v-container>
+  <v-container fluid class="d-flex align-center justify-center my-2">
+    <v-row justify="center">
+      <v-dialog v-model="dialog">
+        <template v-slot:activator="{ props }">
+          <v-btn color="primary" rounded v-bind="props">
+            Frame: {{ imgIndex + 1 }} of {{ framesList.length }}
+          </v-btn>
+        </template>
+        <v-card>
+          <v-card-title>
+            <span class="text-h5"> Select Frame </span>
+          </v-card-title>
+          <v-row class="d-flex align-center justify-center">
+            <v-col v-for="(frame, i) in frames" :key="i" cols="auto">
+              <div v-if="i === imgIndex">
+                <v-img
+                  :src="createUrl(framesUrl, frame)"
+                  :aspect-ratio="16 / 9"
+                  cover
+                  :width="400"
+                  style="
+                    border-style: solid;
+                    border-width: 5px;
+                    border-color: yellowgreen;
+                  "
+                  @click="
+                    imgIndex = i;
+                    dialog = false;
+                  "
+                >
+                  <template v-slot:placeholder>
+                    <v-row
+                      class="fill-height ma-0"
+                      align="center"
+                      justify="center"
+                    >
+                      <v-progress-circular
+                        indeterminate
+                        color="grey-lighten-5"
+                      ></v-progress-circular>
+                    </v-row>
+                  </template>
+                </v-img>
+                <p class="d-flex align-center justify-center">
```

Comment:
This can be simplified, drop the `<p>` element and instead add `class="text-center"` to the `v-row` element